src/config/layers.js

```diff
@@ -431,17 +431,39 @@ export const parks = new TrafimageGeoServerWMSLayer({
   },
 });
 
-export const stuetzpunktbahnhoefe = new HandicapLayer({
+export const stuetzpunktBahnhoefe = new HandicapLayer({
   name: 'ch.sbb.stuetzpunktbahnhoefe',
-  visible: true,
+  visible: false,
   properties: {
+    stutzpunkt: true,
     hasInfos: true,
-    description: 'ch.sbb.stuetzpunktbahnhoefe-desc',
     layerInfoComponent: 'StuetzpunktLayerInfo',
     popupComponent: 'HandicapPopup',
   },
 });
 
+export const barrierfreierBahnhoefe = new HandicapLayer({
+  name: 'ch.sbb.barrierfreierbahnhoefe',
+  visible: true,
+  properties: {
+    barrierfree: true,
+    hasInfos: true,
+    layerInfoComponent: 'BarrierfreierLayerInfo',
+    popupComponent: 'HandicapPopup',
+  },
+});
+
+export const nichtBarrierfreierBahnhoefe = new HandicapLayer({
+  name: 'ch.sbb.nichtbarrierfreierbahnhoefe',
+  visible: true,
+  properties: {
+    barrierfree: false,
+    hasInfos: true,
+    layerInfoComponent: 'NichtBarrierfreierLayerInfo',
+    popupComponent: 'HandicapPopup',
```

Comment:
In the popup of Bern the phone number at "Zusätzliche Informationen" is not linked completely

Reply:
Fixed